packages/vscode-extension/src/qm/vsc_ui.ts

```diff
@@ -556,7 +556,14 @@ export class VsCodeUI implements UserInteraction {
     defaultValue?: string
   ): Promise<Result<SelectFilesResult, FxError>>;
   async selectFileInQuickPick(
-    config: UIConfig<any> & { filters?: { [name: string]: string[] } },
+    config: UIConfig<any> & {
+      filters?: { [name: string]: string[] };
+      possibleFiles?: {
+        id: string;
```

Comment:
What if `id === "default" || id === "browse"`? You may add some checks here?
Also please update the UT of `selectFile`.

Reply:
updated and moved corresponding test file in /qm folder.